### PR TITLE
Make tiles still visible at small screen resolutions

### LINKED ISSUE #43: Scale to All Screen Resolutions
The UI doesn't do a great job scaling to odd screen resolutions.

Some examples:
- 412x975 (Galaxy S20)
- 667x375 (iPhone in landscape)
- 820x1180 (iPad Air)
- 912x1368 (Surface Pro)
- 1200x500

412x975
667x375
820x1180
912x1368
1200x500

Related to #36

### PR DESCRIPTION
**Pull request by wachunga**
Shrinking keyboard is the only way, I think

iphone SE
<table>
<tr>
<th>before</th>
<th>after</th>
</tr>
<tr>
<td>
image
</td>
<td>
image</td>
</tr>
</table>

Together with other PR, I think this closes #43

### CODE REVIEW

File: public/emojis.js [hunk 1]
```diff
@@ -83,6 +83,7 @@ const emojis = {
   CHILD: "👶",
   CIRCUS: "🎪",
   CLAMP: "🗜️",
+  CLAP: "👏",
```
Comment (canadianveggie) [inline public/emojis.js:86]: Sneaky emoji addition. I approve.

File: public/main.css [hunk 2]
```diff
@@ -2,7 +2,7 @@
   --tile-spacing: 4px;
   --total-guesses: 6;
   --max-page-width: 500px;
-  --header-height: 100px;
+  --header-height: 75px;
```
Comment (canadianveggie) [inline public/main.css:5]: Thanks. I was going to make this change.

**Comment by canadianveggie** [comment]
Definitely an improvement.
Screen Shot 2022-02-19 at 12 27 08 PM

There is overlap with IPhone XR in landscape, but I'm ok with that.

**Review by canadianveggie — APPROVED** [review]

File: public/emojis.js (hunk 1 above)
Comment (wachunga) [reply public/emojis.js:86]: Anna's idea!

**Pull request merged**